### ISSUE TITLE
Tutorial update?

### ISSUE THREAD

**Issue by rmib200** [juicycleff/flutter-unity-view-widget#641]
**Describe the bug**

The current instructions on the README could use an update. The video tutorial available is very outdated. I can do it if someone could get into a call with me and clarify a few points. Maybe an update with all the relevant information to start a project from scratch could help a ton of people. 

**Expected behavior**
Having a new tutorial with updated instructions.

**Comment by elvispeter1991** [comment]
Yeah from my side too. Really need an updated tutorial with instruction might be really help full.

**Comment by elvispeter1991** [comment]
Yeah from my side too. Really need an updated tutorial with instruction might be really help full.

**Comment by dawiddszewczyk** [comment]
@juicycleff I can help with improve doc and also close some old issues but I need contributor to do it.

**Comment by juicycleff** [comment]
Will add you @dawiddszewczyk
